sdk/nodejs/runtime/invoke.ts

```diff
@@ -100,3 +127,135 @@ export async function invoke(tok: string, props: Inputs, opts?: InvokeOptions):
         done();
     }
 }
+
+function invokeSync(tok: string, props: any, opts: InvokeOptions = {}): Promise<any> {
+    const label = `Invoking function: tok=${tok} synchronously`;
+    log.debug(label +
+        excessiveDebugOutput ? `, props=${JSON.stringify(props)}` : ``);
+
+    const obj = gstruct.Struct.fromJavaScript(serializePropertiesSync(props));
```

Comment:
we still serialize out all the properties (instead of just calling `fromJavaScript(props)`).  We want to preserve the same serialization logic, and we want to provide an early error if the user includes anything async in the props bag.